src/server/pfs/server/testing/server_test.go

```diff
@@ -5936,7 +5936,6 @@ func TestPFS(suite *testing.T) {
 
 	suite.Run("TestPanicOnNilArgs", func(t *testing.T) {
 		// TODO(2.0 required): Add validation to all PFS endpoints.
```

Comment:
We can probably get rid of this TODO if this test passes now. We still need to do a full pass over the PFS endpoints to move the validation into the validation server and to make sure we are validating all of the fields that should not be unset (or for certain conditions similar to the repo or all flag being set in the delete repo case). We have a card for this in the 2.0 PFS / PPS project, so we should be fine to remove the TODO here.